pkg/odo/util/cmdutils.go

```diff
@@ -184,7 +187,10 @@ func PrintComponentInfo(client *occlient.Client, currentComponentName string, co
 
 			// Let's also get the secrets / environment variables that are being passed in.. (if there are any)
 			secrets, err := client.GetKubeClient().GetSecret(linkedService.SecretName, project)
-			LogErrorAndExit(err, "")
+			if err != nil {
+				log.Error(errors.Cause(err))
```

Comment:
@anandrkskd Why have you added these `log.Error` calls?

Reply:
To log the error we experienced, I will remove it, as I overlooked that in the end it is printed after call telemetry command. I will update it ASAP

Reply:
Updated